### DISCUSSION TITLE
Stac-Browser deployment included in eoapi-k8s

### DISCUSSION THREAD

**Discussion by mdominguezd** [developmentseed/k8s-eoapi#83] [Ideas]
Hi!

I have been testing locally eoapi using the docker compose in [the eoapi repo](https://github.com/developmentseed/eoAPI), and everything works very smoothly when integrating all stac-browser, titiler-pgstac, stac-api and the pgstac database. However, I see that on the [eoapi-k8s reoi](https://github.com/developmentseed/eoapi-k8s) the stac-browser service is not deployed. I was wondering. Why is that? Should Stac-Browser be deployed independently?

**Comment by ranchodeluxe** [comment]
Sorry @mdominguezd for the late reply

Looks like someone else had the need and is close to a solution: https://github.com/developmentseed/eoapi-k8s/issues/62#issuecomment-2229099746
